org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/dialogs/AbapGitStagingCredentialsDialog.java

```diff
@@ -32,12 +32,10 @@
 	private Label storeCredentialCheckBoxLabel;
 
 	private boolean storeCredsInSecStore = false;
-	private IExternalRepositoryInfoRequest repoCredentials;
 
-	public AbapGitStagingCredentialsDialog(Shell parentShell, String errorMessage, IExternalRepositoryInfoRequest repositoryCredentials) {
+	public AbapGitStagingCredentialsDialog(Shell parentShell, String errorMessage) {
 		super(parentShell);
 		this.errorMessage = errorMessage;
-		this.repoCredentials = repositoryCredentials;
```

Comment:
In earlier version, the constructor accepted `credentials` as parameter as well, that was retrieved from `Secure Storage` elsewhere. These credentials are then set into the credentials dialog. But this leads to potential security risk, because retrieving credentials from credentials dialog (user entered creds) which is a public method, can also indirectly help retrieve creds from Secure Storage.